Source/WebCore/testing/Internals.cpp

```diff
@@ -3203,7 +3209,13 @@ String Internals::serviceWorkerClientInternalIdentifier(const Document& document
 
 RefPtr<WindowProxy> Internals::openDummyInspectorFrontend(const String& url)
 {
-    auto* inspectedPage = contextDocument()->frame()->page();
+    if (!contextDocument())
```

Comment:
Seems like this should go into a local variable the way it does in `setInspectorIsUnderTest`.

Reply:
Done